### ISSUE TITLE
Duplicated snippets in toolkit

### ISSUE THREAD

**Issue by GrahamStuart** [Komodo/KomodoEdit#3084]
### Short Summary

Duplicated Abbrevations

### Steps to Reproduce

I have IDE 10 and Edit 11 RC3 installed at the same time. 
In Edit 11's toolbox, I open the Abbreviations 
and look at any of the Python, Ruby, XUL, and Stackato sets

### Expected results

IDE 10 only shows one example for each Abbreviation

### Actual results

Edit 11 shows two identical versions of each Abbreviation

### Platform Information

Edit 11 RC and Komodo IDE 10.2.3
OS X 10.12.4

### Additional Information

ActivePython 3.5.3 installed, Python3 set as default file type with .py files mapped to Python3. (probably not important since the issue is wider than Python).

-->

**Comment by Naatan** [comment]
When you right click the Abbreviations folder and select "Show in File Manager", do the files occur twice in your file manager also?

**Comment by GrahamStuart** [comment]
Not exactly. When I looked with Finder I see that the ones which are duplicates have two files slightly different files. One has .ktf extension and the other has .komodotool
eg:
main.ktf
meth.komodotool
meth.ktf
When viewed in the Left Column viewer I would see one entry for main and two for meth.

**Comment by Naatan** [comment]
@GrahamStuart would you mind moving your 11.0 profile folder aside and running Komodo 11 again, then verify that the issue happened again and share the pystderr.log file from the newly created 11.0 profile folder.

Thanks!

**Comment by GrahamStuart** [comment]
Fresh Profile has the same thing. Having a bit of a challenge getting the log file off the VM.....

**Comment by Naatan** [comment]
You could just pastebin it from the VM

**Comment by GrahamStuart** [comment]
[pystderr.txt](https://github.com/Komodo/KomodoEdit/files/1304450/pystderr.txt)

**Comment by Naatan** [comment]
Nothing useful in there unfortunately. I just tried upgrading my profile on edit and it went fine. 

@mjainQA can you attempt this on macOS? Make sure you have a 10.* profile folder and not an 11.0 profile folder, then launch Komodo Edit and check if you have double snippets in your toolbox.
